docs/modules/plugins/pages/plugin-visual.adoc

```diff
@@ -284,8 +289,12 @@ where:
 |The size of right part of the final screenshot to cut.
 
 |`mobile.screenshot.strategy.<YOUR_STRATEGY_NAME>.shooting-strategy`
-|`SIMPLE`
-|Screenshot shooting strategy name (`SIMPLE` by default)
+|`VIEWPORT`, `FULL_SCREEN`, `SIMPLE`
+a|Screenshot shooting strategy name (`VIEWPORT` by default)
+[WARNING]
+====
+The shooting strategy `SIMPLE` is deprecated and will be removed in VIVIDUS 0.6.0. Use `VIEWPORT` instead. `VIEWPORT` is equal to `SIMPLE` shooting strategy
```

Comment:
The shooting strategy `SIMPLE` is deprecated and will be removed in VIVIDUS 0.6.0. Use `VIEWPORT` instead. `VIEWPORT` is equal to `SIMPLE` shooting strategy